server/modules/category/categoryValidator.js

```diff
@@ -74,7 +74,7 @@ const _extraPropValidators = {
     Validator.validateNumber(Validation.messageKeys.categoryEdit.itemExtraPropInvalidNumber, { key })(key, extra),
   [Category.itemExtraDefDataTypes.geometryPoint]: (key, extra) => {
     const point = Point.parse(extra[key])
-    return GeoUtils.isCoordinateValid(Point.getSrs(point), Point.getX(point), Point.getY(point))
+    return point && GeoUtils.isCoordinateValid(Point.getSrs(point), Point.getX(point), Point.getY(point))
       ? null
       : ValidationResult.newInstance(Validation.messageKeys.categoryEdit.itemExtraPropInvalidGeometryPoint, { key })
   },
```

Comment:
should we use for readability in this kind of cases
```
if (condition) return null
return ValidationResult.newInstance(...)
```

Reply:
ok, in any case this part of the code will be changed with the use of arena-core modules.